xo.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  space: true,
+  semicolon: false,
```

Comment:
@cossssmin Not sure if you should not use a semicolon, let the optimizers do it - the cat should be readable

Reply:
OK, will refactor to use semicolons 👍